File: LeetCode/Python/0412. Fizz Buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        
        for i in range(1, n+1):
            if not i % 5 and not i % 3:
                answer.append("FizzBuzz")
            elif not i % 3:
                answer.append("Fizz")
            elif not i % 5:
                answer.append("Buzz")
            else:
                answer.append(str(i))
            
        return answer

File: LeetCode/Java/0412. Fizz Buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        ArrayList<String> answer = new ArrayList<>();
        String s;
        
        for (int i = 1; i <= n; i++) {
            if (i % 3 == 0 && i % 5 == 0)
                s = "FizzBuzz";
            else if (i % 3 == 0)
                s = "Fizz";
            else if (i % 5 == 0)
                s = "Buzz";
            else
                s = String.valueOf(i);
            answer.add(s);
        }
        
        return answer;
    }
}
